draft-ietf-dprive-dnsoquic.md

```diff
@@ -335,10 +338,10 @@ a QUIC Stop Sending with error code DOQ_REQUEST_CANCELLED. This may be sent at
 any time but will be ignored if the server has already sent the response. The
 corresponding DNS transaction MUST be abandoned.
 
-A server that receives STOP_SENDING MUST issue a RESET_STREAM with error code
-DOQ_REQUEST_CANCELLED, unless it has already sent a complete response in which
-case it MAY ignore the STOP_SENDING request. Servers MAY limit the number of
-DOQ_REQUEST_CANCELLED errors received on a connection before choosing to close
+A server that receives STOP_SENDING MUST issue a RESET_STREAM request.
+The error code of the reset stream request SHOULD be sent to 
+DOQ_REQUEST_CANCELLED. Servers MAY limit the number of
+reset stream requests received on a connection before choosing to close
 the connection.
```

Comment:
This text is unclear. RESET_STREAM is an action by the sender, not a request for action.

With the change, you just seem to be restating the requirements of https://www.rfc-editor.org/rfc/rfc9000.html#section-3.5-4.

It seems the text is suggesting that a server might want to protect itself from gross cancellations, so you might want to rephrase it a little.

How about this instead?

```suggestion
Servers that receive STOP_SENDING act in accordance with {{Section 3.5 of RFC9000}}.
Servers MAY impose implementation limits on the total number or rate of request cancellations.
If limits are encountered, servers SHOULD close the connection with error code DOQ_EXCESSIVE_LOAD 
```